gp-includes/route.php

```diff
@@ -31,9 +31,30 @@ public function __construct() {
 		add_action( 'gp_before_request', array( $this, 'check_uri_trailing_slash' ) );
 	}
 
-	public function die_with_error( $message, $status = 500 ) {
+	/**
+	 * Shows a template and exit.
+	 *
+	 * @param string $message  The message to display.
+	 * @param int    $status   The HTTP status code.
+	 * @param string $title    The title of the page.
+	 * @param string $template The template to use.
+	 *
+	 * @return void
+	 */
+	public function die_with_error( string $message, int $status = 500, string $title = '', string $template = 'error' ) {
```

Comment:
See below, to avoid displaying `< GlotPress` in the title, we could also do this (which I'd actually prefer since it also gives the text "Error" to the headline).
```suggestion
	public function die_with_error( string $message, int $status = 500, string $title = false, string $template = 'error' ) {
		if ( ! $title ) {
			$title = __( 'Error', 'glotpress' );
		}
```

Reply:
I have updated it, allowing a null value as default for the $title parameter.